pkg/build/api/v1/types.go

```diff
@@ -55,7 +55,7 @@ type BuildStatus struct {
 	Phase BuildPhase `json:"phase" description:"observed point in the build lifecycle"`
 
 	// Cancelled describes if a cancelling event was triggered for the build.
-	Cancelled bool `json:"cancelled,omitempty" description:"describes if a canceling event was triggered for the build"`
+	Cancelled bool `json:"cancelled,omitempty" description:"describes if a cancelling event was triggered for the build"`
```

Comment:
You most probably need to update swagger for this change

./hack/update-generated-swagger-spec.sh